lib/internal/bootstrap/node.js

```diff
@@ -265,6 +247,20 @@ Object.defineProperty(process, 'features', {
   }
 });
 
+{
+  const {
+    fatalException,
+    setUncaughtExceptionCaptureCallback,
+    hasUncaughtExceptionCaptureCallback
+  } = NativeModule.require('internal/process/execution');
+
+  process._fatalException = fatalException;
+  process.setUncaughtExceptionCaptureCallback =
+    setUncaughtExceptionCaptureCallback;
+  process.hasUncaughtExceptionCaptureCallback =
+    hasUncaughtExceptionCaptureCallback;
+}
+
 // User-facing NODE_V8_COVERAGE environment variable that writes
```

Comment:
The higher-order bit is that everything below this point are environment-dependent and should be moved out of node.js - then it would be possible to snapshot Node.js after the execution of this script.